assets/templates/alertgroup.html

```diff
@@ -308,3 +312,14 @@
     <%- label.text %>
   </<%= elem %>>
 </script>
+
+<script type="application/json" id="alert-annotation">
+<% var cls = "" %>
```

Comment:
`cls` is used to set `hidden hidden-annotation` if needed, not sure how `<details>` and `<summary>` relates to that